telemetry/receivedenevlope.go

```diff
@@ -0,0 +1,62 @@
+package telemetry
+
+import (
+	"database/sql"
+	"time"
+)
+
+type ReceivedEnvelope struct {
+	ID              int    `json:"id"`
+	MessageHash     string `json:"messageHash"`
+	SentAt          int64  `json:"sentAt"`
+	CreatedAt       int64  `json:"createdAt"`
+	PubsubTopic     string `json:"pubsubTopic"`
+	Topic           string `json:"topic"`
+	ReceiverKeyUID  string `json:"receiverKeyUID"`
+	NodeName        string `json:"nodeName"`
+	ProcessingError string `json:"processingError"`
+}
+
+func (r *ReceivedEnvelope) put(db *sql.DB) error {
+	r.CreatedAt = time.Now().Unix()
+	stmt, err := db.Prepare(`INSERT INTO receivedEnvelopes (messageHash, sentAt, createdAt, pubsubTopic,
+							topic, receiverKeyUID, nodeName, processingError)
+							VALUES ($1, $2, $3, $4, $5, $6, $7, $8)
+							ON CONFLICT ON CONSTRAINT receivedEnvelopes_unique DO NOTHING
```

Comment:
I noticed that when we start the observed Status app, the service receives envelopes which were already added. This breaks the constraint and there are errors. I decided to add "DO NOTHING" in case of conflict. This way there is no error.